app/src/main/java/net/squanchy/notification/NotificationCreator.java

```diff
@@ -109,6 +109,14 @@ private PendingIntent createPendingIntentForSingleEvent(String eventId) {
         return taskBuilder.getPendingIntent(0, PendingIntent.FLAG_CANCEL_CURRENT);
     }
 
+    private String getPlaceName(Event event) {
```

Comment:
This can also be done with

`event.place.map(place -> place.name).or(EMPTY_PLACE_NAME)`